Tests/InstallationTests/CocoapodsInstallation/Gemfile.lock

```diff
@@ -32,11 +32,11 @@ GEM
     aws-sigv4 (1.4.0)
       aws-eventstream (~> 1, >= 1.0.2)
     babosa (1.0.4)
-    claide (1.0.3)
-    cocoapods (1.12.1)
+    claide (1.1.0)
+    cocoapods (1.13.0)
```

Comment:
Needed to update it here too.

Reply:
Great update! 👏 Excited to see the addition of visionOS to the deployment targets in the Podspec. Thanks for keeping the Swift version up-to-date and providing clear instructions on the required Xcode version. The integration of the visionOS CocoaPods lint check and the switch to Xcode 15 beta 8 for the CI job make it even better.

However, I noticed the mention of the issue CocoaPods/CocoaPods#11965. Any insights on how this is being addressed or any potential workarounds? Looking forward to the continued progress! 🚀

Reply:
Support for it on CocoaPods is still not working: https://github.com/CocoaPods/CocoaPods/pull/12159#issuecomment-1863576353
We haven't received an update from them, so until then unfortunately we can't release this.